frontend/packages/helm-plugin/src/catalog/providers/useHelmCharts.tsx

```diff
@@ -25,11 +38,11 @@ const useHelmCharts: CatalogExtensionHook<CatalogItem[]> = ({
   }, []);
 
   const normalizedHelmCharts: CatalogItem[] = React.useMemo(
-    () => normalizeHelmCharts(helmCharts, namespace, t),
-    [helmCharts, namespace, t],
+    () => normalizeHelmCharts(helmCharts, chartRepositories, namespace, t),
+    [chartRepositories, helmCharts, namespace, t],
   );
 
-  const loaded = !!helmCharts;
+  const loaded = !!helmCharts && chartRepositoriesLoaded;
```

Comment:
Yes, Don't want to hold the catalog items if chart repositories don't load up because of some error. Repositories only add value to the title of the chart repositories but still fallback to index title.